src/UnitTestDatabase.cpp

```diff
@@ -0,0 +1,56 @@
+#include "Database.h"
+#include <gtest/gtest.h>
+#include <thread>
+shortlink::Database database;
+
+void testDatabaseCreate(std::string raw_url) {
```

Comment:
Either put this in an anonymous namespace or use the test fixture.

Please use const for input argument.

Please fix function naming, first letter should be capital.
https://google.github.io/styleguide/cppguide.html#Function_Names